cps/templates/modal_restriction.html

```diff
@@ -21,7 +21,7 @@ <h4 class="modal-title hidden" id="h4">{{_('Select allowed/denied Custom Column
           </table>
           <form id="add_restriction" action="" method="POST">
           <div class="form-group required">
-            <label for="add_element">{{_('Add View Restriction')}}</label>
+            <label for="add_element">{{_('Add New Restriction')}}</label>
```

Comment:
I wanted to express that the restrictions are for the View/visibility of books, maybe "Add New Visibility Restriction" is a good description?